packages/cli/lib/validators/marketplaceValidators/templateLimitValidator.js

```diff
@@ -0,0 +1,26 @@
+const { VALIDATION_RESULT } = require('../constants');
+
+const VALIDATOR_NAME = 'TemplateLimitValidator';
+const TEMPLATE_LIMIT = 50;
+const TEMPLATE_REGEX = new RegExp(/.html+$/);
```

Comment:
Will this match `module.html`?

Reply:
Yeah that's a good catch, updated in ee4a6dd